google/cloud/storage/blob.py

```diff
@@ -1007,6 +1016,18 @@ def _do_multipart_upload(
         :type predefined_acl: str
         :param predefined_acl: (Optional) Predefined access control list
 
+        :type if_generation_match: long
+        :param if_generation_match: (Optional) Precondition control if generation match
+
+        :type if_generation_not_match: long
+        :param if_generation_not_match: (Optional) Precondition control if generation not match
+
+        :type if_metageneration_match: long
+        :param if_metageneration_match: (Optional) Precondition control if metageneration match
+
+        :type if_metageneration_not_match: long
+        :param if_metageneration_not_match: (Optional) Precondition control if metageneration not match
```

Comment:
Missed in first pass @IlyaFaer, could update documentation to reflect what's stated in cloud storage API reference documentation:

https://cloud.google.com/storage/docs/json_api/v1/objects/insert

Reply:
Okay, pushed the comments very similar to those in docs